src/ru/sberbank/javaSchool/transport/Motorcycle.java

```diff
@@ -0,0 +1,9 @@
+package ru.sberbank.javaSchool.transport;
+
+import java.util.GregorianCalendar;
+
+public class Motorcycle extends Transport {
+    public Motorcycle(String vin, GregorianCalendar dateOfIssue, int maxSpeed, int length, int width, int height, TypeTransport typeTransport) {
```

Comment:
you don't need set TypeTransport as constructors argument, because is MOTORCYCLE always. It's true for another transport types